sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/browserless/build.sbt

```diff
@@ -6,8 +6,8 @@ scalaVersion := "2.11.8"
 
 scalaJSUseMainModuleInitializer := true
 
-// Adds a dependency on the node-uuid npm package
-npmDependencies in Compile += "node-uuid" -> "1.4.7"
+// Adds a dependency on the uuid npm package
+npmDependencies in Compile += "uuid" -> "3.1.0"
```

Comment:
The uuid changes have been extracted to https://github.com/scalacenter/scalajs-bundler/pull/167